CHANGELOG.rst

```diff
@@ -9,11 +9,16 @@
 * Add ``buffer`` attribute to stdin stub class ``pytest.capture.DontReadFromInput``
   Thanks `@joguSD`_ for the PR.
 
+* Fix UnicodeEncodeError when string comparison with unicode is failed. (`#1864`_)
```

Comment:
Nitpick: "has failed" or "fails" :grin: 

Also: Use double backticks around `UnicodeEncodeError` so it appears in a monospaced font (see `buffer` on the item above this one).